### PR TITLE
Nuclear Missile Buff

### PR DESCRIPTION
**Pull request by BZENDSKY**
Adds my plugin called Nuclear Missile Buff.

### CODE REVIEW

**Review by MCOfficer — APPROVED** [review]
No icon?

**Comment by BZENDSKY** [comment]
There was one in the repository. How do I upload that here?

**Comment by BZENDSKY** [comment]
Does this work?

File: manifests/Nuclear Missile Buff.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Nuclear Missile Buff
+authors: BZENDSKY
+homepage: https://github.com/BZENDSKY/Nuclear-Missile-Buff
+license: GPL-3.0-or-later
+version: v1.0
+shortDescription: Buffs the Nuclear Missile.
+description: Gives the Nuclear Missile more shield and hull damage.
+url: https://github.com/BZENDSKY/Nuclear-Missile-Buff/archive/refs/tags/v1.0.zip
+iconUrl: https://github.com/BZENDSKY/Nuclear-Missile-Buff/raw/nuclear/icon.png
```
Comment (MCOfficer) [inline manifests/Nuclear Missile Buff.yaml:9]: Version this one as well, please:
```suggestion
iconUrl: https://github.com/BZENDSKY/Nuclear-Missile-Buff/raw/v1.0/icon.png
```

File: manifests/Nuclear Missile Buff.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+autoupdate:
+  type: tag
+  url: https://github.com/BZENDSKY/Nuclear-Missile-Buff/archive/refs/tags/$version.zip
+  iconUrl: https://raw.githubusercontent.com/BZENDSKY/Nuclear-Missile-Buff/nuclear/icon.png
```
Comment (MCOfficer) [inline manifests/Nuclear Missile Buff.yaml:13]: and likewise
```suggestion
  iconUrl: https://raw.githubusercontent.com/BZENDSKY/Nuclear-Missile-Buff/$tag/icon.png
```

**Comment by BZENDSKY** [comment]
Wow, that showed up fast on the plugin page.

**Pull request merged**